frontend/src/modules/signup/stakeholderSchema.js

```diff
@@ -118,35 +79,15 @@ export const schema = {
         id: "tabs",
         value: ["S2"],
       },
-      required: [
-        "orgName",
-        "orgRepresentative",
-        "orgDescription",
-        "orgUrl",
-        "orgLogo",
-      ],
+      required: ["orgName"],
       properties: {
         orgName: {
-          title: "What is the name of your organisation?",
+          title: "Search Your Entity",
           type: "string",
         },
-        orgRepresentative: {
-          title: "Which representative groups fits your organisation?",
+        memberName: {
+          title: "Is your Entity not a GPML Member yet?",
           type: "string",
```

Comment:
I'm not sure, this field will be a select dropdown or string

Reply:
WIP for the next PR